packages/manager/modules/pci/src/projects/project/users/users.constants.js

```diff
@@ -1,5 +1,7 @@
 export const HORIZON_LINK = 'https://horizon.cloud.ovh.net/openstackdashboard?username={username}';
+export const STRING_ONLY_PATTERN = /^[a-zA-Z]+$/;
 
 export default {
   HORIZON_LINK,
+  STRING_ONLY_PATTERN,
```

Comment:
this name doesn't describe the regex very well, maybe something like ALPHA_CHARACTERS_REGEX would be a better?